pkgs/development/python-modules/soundcard/default.nix

```
@@ -0,0 +1,49 @@
+{
+  buildPythonPackage,
+  cffi,
+  fetchFromGitHub,
+  lib,
+  libpulseaudio,
+  numpy,
+  setuptools,
+  testers,
+}:
+
+buildPythonPackage rec {
+  pname = "soundcard";
+  version = "0.4.3";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "bastibe";
+    repo = pname;
```

This is a bit of a nit, but nowadays it's better practice not to use `pname` for substituting occurrences of the package name.

```suggestion
    repo = "soundcard";
```